3-arrays/properties and methods of arrays.js

```diff
@@ -0,0 +1,64 @@
+// Some properties and methods of array are--
```

Comment:
```suggestion
// Some properties and methods of array are :
```
You should explain what is a property and a method, the syntax of them